test/spec/modules/appnexusAstBidAdapter_spec.js

```diff
@@ -1,194 +1,165 @@
 import { expect } from 'chai';
 import { spec } from 'modules/appnexusAstBidAdapter';
 import { newBidder } from 'src/adapters/bidderFactory';
-import bidmanager from 'src/bidmanager';
 
 const ENDPOINT = '//ib.adnxs.com/ut/v3/prebid';
 
-const REQUEST = {
```

Comment:
Also... shouldn't these tests be updated in master? There's nothing 1.0 or concurrent-auctions about them.

Did we give up on making those centralized tests so that adapter-specific ones only need to define JSON files?

Reply:
Yes, tests can be updated for master as well. I will create a PR for that. 

Plan for JSON file based tests is still on. I will be adding that pretty soon.